predict.py

```diff
@@ -48,24 +33,25 @@ def load_model(training_dir, device):
     return model.eval(), num_classes
 
 
-def get_dataset_properties(dataset_ids):
+def get_dataset_labels(dataset_ids):
     datasets_client = DatasetsClient()
-    props = {}
+    labels = []
     for dataset_id in dataset_ids:
         dataset = datasets_client.get_dataset(dataset_id)
-        props = dataset.props
+        labels = dataset.props['categories'][0]['labels']
         break
-    return props
+    return labels
 
 
 def decode_segmap(out, label_colors):
     om = torch.argmax(out.squeeze(), dim=0).detach().cpu().numpy()
+    print(om)
```

Comment:
please remove or add description for `om`

Reply:
Adding description might be better because this value is argmaxed prediction result.

Reply:
Updated -> 83c2f9a